aem/core_aem/README.md

```diff
@@ -18,28 +18,22 @@ service looks up the following OSGI configuration keys:
 * `aio.consumer.org.id`  your Adobe Developer Console consumer orgnaization id (`project.org.id`)
 * `aio.ims.org.id` your Adobe Developer Console IMS Organization ID (`project.org.ims_org_id`)
 * `aio.workspace.id` your Adobe Developer Console workspace Id (`project.workspace.id`)
-* `aio.credential.id` your Adobe Developer Console jwt credential id (`project.workspace.details.credentials[i].id`)
-* `aio.api.key` your Adobe Developer Console jwt credential API Key (or Client ID) (`project.workspace.details.credentials[i].jwt.client_id`)
-* `aio.client.secret` your Adobe Developer Console jwt credential client secret (`project.workspace.details.credentials[i].jwt.client_secret`)
-* `aio.meta.scopes` a comma separated list of metascopes associated with your API, see your Adobe Developer Console jwt credential metascopes (`project.workspace.details.credentials[i].jwt.meta_scopes`)
-* `aio.technical.account.id` your Adobe Developer Console jwt credential technical account id (`project.workspace.details.credentials[i].jwt.technical_account_id`)
-* `aio.encoded.pkcs8` your private key (in a base64 encoded pkcs8 format) see below
+* `aio.api.key` your Adobe Developer Console credential API Key (or Client ID) 
 
+When using JWT credentials also set
+* `aio.credential.id` your Adobe Developer Console jwt credential id
+* `aio.client.secret` your Adobe Developer Console jwt credential client secret 
```

Comment:
its no longer JWT. it is OAuth S2S credential

Reply:
as line 23 states, this list is still `When using JWT credentials`